test/parallel/test-repl-tab-complete.js

```diff
@@ -443,13 +433,12 @@ const warningRegEx = new RegExp(
         Buffer.alloc(0) :
         new type(0));
 
+    assert.strictEqual(data[0].includes('ele.biu'), true);
```

Comment:
It would be great to have tests that specify the behaviour for long arrays better.